aft.yaml

```diff
@@ -13,6 +13,36 @@ dependencies:
   json_serializable: 6.3.1
   uuid: 3.0.6
 
-# Packages to ignore in all repo operations
+# Packages to ignore in all repo operations.
 ignore:
   - synthetic_package
+
+# Strongly connected components which should have major version bumps happen 
```

Comment:
I've kind of changed the meaning of the word minor/major version throughout. Since true "major" version bumps, e.g. `0.x -> 1.x` are so insanely rare and would not be handled by this script, I've repurposed breaking/major to be useful descriptions of, for example, making a breaking change to a low-level API or adding a new category, etc. where all the packages should be bumped in unison to the next version (`0.5.x -> 0.6.0` or `1.0.x -> 1.1.0`).

In this way, minor version bumps constitute everything else - for which they only apply to the affected packages. Changing a README in aws_common, for example, would be a "minor" version bump of just aws_common (e.g. `0.1.5` -> `0.1.6`).

Reply:
Okay. Does that mean amplify_auth_cognito could be on version 1.1.8 and amplify_datastore could be on version 1.1.7? I'm not clear if you are suggesting we change how we pin versions in amplify or not.

Reply:
Correct. I'm suggesting we don't pin versions anymore since there's no real need - component packages would always be at the same SemVer minor version though. The only thing that would change is the SemVer patch version.